rmw_dds_common/src/security.cpp

```diff
@@ -40,20 +40,27 @@ bool get_security_files(
   };
 
   for (const std::pair<const std::string, std::string> & el : required_files) {
-    rcpputils::fs::path full_path(secure_root);
+    std::filesystem::path full_path(secure_root);
     full_path /= el.second;
-    if (!full_path.is_regular_file()) {
+    std::filesystem::directory_entry target(full_path);
+    if (!target.is_regular_file()) {
```

Comment:
Rather than constructing this `directory_entry`, can we just call the free function?

```suggestion
    if (!std::filesystem::is_regular_file(full_path)) {
```

Reply:
[std::filesystem::path](https://en.cppreference.com/w/cpp/filesystem/path) does not have an equivalent function to  is_regular_file.
Actually, std::filesystem::path does not case about the path is valid or not.
So, we need to check it in another way.

Do you have any thoughts?